flipperdevices/flipperzero-firmware
# New toolchain with gcc 12

# What's new

- Updated toolchain version to incorporate gcc 12 and updated Python dependecies
- Cleaned up a few warnings 
- Added FURI_DEPRECATED macro 

# Verification 

- Build & run fw
- Try marking something in system headers with FURI_DEPRECATED and see build logs

# Checklist (For Reviewer)
